web/themes/new_weather_theme/templates/layout/page--point.html.twig

```diff
@@ -59,14 +59,33 @@
           {% include '@new_weather_theme/partials/radar.html.twig' with { 'point': weather.point  } %}
 
           {% include '@new_weather_theme/partials/satellite.html.twig' %}
-
-            {#{ drupal_block("weathergov_area_forecast_discussion") }#}
         </div>
 
          <div class="wx-tab-container wx-focus-offset-205" id="daily" role="tabpanel" aria-labelledby="daily-tab-button" tabindex="0">
           <h2 class="usa-sr-only">7 days</h2>
           {{ drupal_block("weathergov_weather_story") }}
 
+          <div class="margin-x-2 tablet:margin-x-auto tablet:grid-container margin-bottom-4 padding-x-0 tablet:padding-x-2">
+            <div class="grid-row">
+              <div class="grid-col-12 tablet-lg:grid-offset-2 tablet-lg:grid-col-8">
+                <div class="bg-accent-cool-lightest border border-accent-cool-lighter padding-105" role="alert">
+                  <svg
+                    class="usa-icon height-3 width-3 text-accent-cool-dark margin-right-1"
+                    aria-hidden="true"
+                    focusable="false"
+                    role="img"
+                    style="vertical-align: middle;"
+                  >
+                    <use
+                      xlink:href="/{{ directory }}/assets/images/uswds/sprite.svg#info_outline"
+                    ></use>
+                  </svg><a href="/afd/{{ weather.grid.wfo }}">Current Area Forecast Discussion</a>
```

Comment:
Two small things on the link:
1. We should add a "usa-link" class to the link so that it gets the right colors
2. For this instance of the link text I used just "Area Forecast Discussion" without the word "Current". Noting because then it doesn't wrap on the smallest screens.

Reply:
You know what's annoying about that second bullet? I did that, and I lost the change when I accidentally messed up something else and I guess I ctrl-z'ed too much.